Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    notion-ruby-client (0.1.0.pre.beta1)
+    notion-ruby-client (0.1.0.pre.beta2)
```

Comment:
I’m not entirely sure why, but the test suite is failing at the `bundle install` step. Could you try and remove all changes from `Gemfile.lock` from your PR?